README.md

```diff
@@ -102,7 +102,7 @@ Please read the [Organisation's Code of Conduct](https://github.com/kube-logging
 
 *For more information, read the [developer documentation](https://kube-logging.github.io/docs/developers)*.
 
-[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7941/badge)](https://www.bestpractices.dev/projects/7941)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/7941/badge?cache=20231206)](https://www.bestpractices.dev/projects/7941)
```

Comment:
Could you move this to the header (next to the other badges)?

Reply:
sure, just did